packages/core/test/callout/calloutTests.tsx

```diff
@@ -29,13 +29,6 @@ describe("<Callout>", () => {
         assert.isTrue(wrapper.hasClass(Classes.INTENT_DANGER));
     });
 
-    it("spreads HTML props", () => {
```

Comment:
why remove this test?

Reply:
I didn't find it very useful, and it was using invalid HTML props